npm build fails `sh: microbundle: command not found`

I get the following error when I try to build the project:

```
[heumos@wave tubeify]$ npm run build

> tubeify@1.0.0 build /home/heumos/git/tubeify
> microbundle

sh: microbundle: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! tubeify@1.0.0 build: `microbundle`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the tubeify@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/heumos/.npm/_logs/2019-10-09T08_20_15_958Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -8,6 +8,7 @@ Tubeify can use as both a CLI tool and a JavaScript library.
 
 ```bash
 npm install -g csv2json
+npm install
 csv2json -t tests/test.tsv  > tests/test.json
 npm run build
 node cli.js -j tests/test.json -b 25 -l 1000 -t -1

